.github/workflows/linux.yml

```diff
@@ -275,6 +293,54 @@ jobs:
       image: 'openvinogithubactions.azurecr.io/dockerhub/ubuntu:20.04'
       affected-components: ${{ needs.smart_ci.outputs.affected_components }}
 
+  JS_API:
+    name: JS API
+    needs: [ Build, Smart_CI ]
+    defaults:
+      run:
+        shell: bash
+    runs-on: 'aks-linux-4-cores-16gb'
+    container:
+      image: 'openvinogithubactions.azurecr.io/dockerhub/ubuntu:20.04'
+    env:
+      OPENVINO_JS_DIR: ${{ github.workspace }}/openvino/src/bindings/js
+      OPENVINO_JS_LIBS_DIR: ${{ github.workspace }}/openvino/src/bindings/js/node/bin
+      NODE_VERSION: 18
```

Comment:
Should different Node versions be tested? @vishniakov-nikolai

Reply:
There is no plan for now, thanks